src/app/app.server.ts

```diff
@@ -0,0 +1,39 @@
+import express, { Application } from 'express';
+import Logger from '../utils/logger';
+
+class App {
```

Comment:
I know the approach of the OOP is really nice when you are trying to encapsulate some functionality, but in this case, since we are creating a single server for the backend there's no need to create a class.

Sometimes it's better to leave our solutions as simple as possible. 😊

Reply:
I am experimenting with this approach since I've never done it before. Can I keep it? 

Tho, if you definitely consider it as not suitable then I'll change it back.